src/misc/setup.cpp

```diff
@@ -1151,6 +1167,16 @@ bool CommandLine::FindInt(char const * const name,int & value,bool remove) {
 	return true;
 }
 
+bool CommandLine::FindIntOption(const std::string &name, int &value, bool remove) {
+	cmd_it it,it_next;
+	if (!IsOption(name, it))
+		return false;
+	it_next=it;++it_next;
+	value=atoi((*it_next).c_str());
+	if (remove) cmds.erase(it,++it_next);
```

Comment:
If the prior assert ensures that `it_next` isn't `end()`, then this increment should be OK even if  `if_next` lands on `end()` (because `erase` erases up to but not including the last element).